### PR TITLE
chore: 1.0.0-beta.126

### PR DESCRIPTION
**Pull request by IgorKarpiuk**
## What/Why/How?

Release 1.0.0-beta.126:
- https://github.com/Redocly/redocly-cli/pull/1045
- https://github.com/Redocly/redocly-cli/pull/1046
- https://github.com/Redocly/redocly-cli/pull/1051
- https://github.com/Redocly/redocly-cli/pull/1053
- https://github.com/Redocly/redocly-cli/pull/1055
- https://github.com/Redocly/redocly-cli/pull/1057
- https://github.com/Redocly/redocly-cli/pull/1067
- https://github.com/Redocly/redocly-cli/pull/1070
- https://github.com/Redocly/redocly-cli/pull/1071

## Check yourself

- [x] Code is linted
- [x] Tested with redoc/reference-docs/workflows
- [x] All new/updated code is covered with tests

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -5,6 +5,26 @@ toc:
 
 # Redocly CLI changelog
 
+## 1.0.0-beta.126 (2023-05-09)
+
+### Features
+
+- Added new options for join command: `--decorate` and `--preprocess`
```
Comment (tatomyr) [inline docs/changelog.md:12]:
```suggestion
- Added new options for the `join` command: `--decorate` and `--preprocess`.
```

File: docs/changelog.md [hunk 2]
```diff
@@ -5,6 +5,26 @@ toc:
...
+
+### Fixes
+
+- Renamed `required-min-length-string-type-property` to `required-string-property-missing-min-length`
+- Fixed a bug where context.resolve() did not work when applied to nodes belonging to external oas (x-) attributes that contain an array.
+- Don't execute decorators and preprocessors in join command
+- Display an error if apis or rules for the `lint` command not provided
+- Display an error if the api path refer to a folder
+- Fixed the issue with `push` command doesn't recognize API definitions with a spaces
```
Comment (tatomyr) [inline docs/changelog.md:21]:
```suggestion
- Fixed a bug with external OAS (x-) attributes containing an array.
- Display an error if the api path refers to a folder.
- Fixed the `push` command not recognising API definitions with spaces.
```

File: docs/changelog.md [hunk 3]
```diff
@@ -5,6 +5,26 @@ toc:
...
+### Fixes
+
+- Fixed a bug with external OAS (x-) attributes containing an array.
+- Display an error if the api path refers to a folder.
+- Fixed the `push` command not recognising API definitions with spaces.
+
+### Changes
+
+- Display an error if apis or rules for the `lint` command are not provided.
+- Renamed `required-min-length-string-type-property` rule to `required-string-property-missing-min-length`.
```
Comment (tatomyr) [inline docs/changelog.md:23]: Please don't forget to update the docs.

File: docs/commands/lint.md [hunk 4]
```diff
@@ -106,6 +106,13 @@ redocly lint --config=./another/directory/config.yaml
 
 The `--extends` option allows you to extend the existing configuration. This option accepts one of the following values: `minimal`, `recommended`, `all`. Each of the values is a base set of rules that the lint command will use. You can further modify this set in cases when you want to have your own set of rules based on the existing one, including particular rules that cover your specific needs.
 
+:::warning Important
+
+When you run the `lint` command without a configuration file, it will use the `extends: recommended` by default.
+However, if you have a configuration file, but it doesn't include any rules or extensions, the `lint` command will show an error.
```
Comment (adamaltman) [inline docs/commands/lint.md:112]:
```suggestion
When you run the `lint` command without a configuration file, it uses the `extends: recommended` by default.
However, if you have a configuration file, but it doesn't include any rules or extends configuration, the `lint` command shows an error.
```

**Pull request merged**